test/pylib/util.py

```diff
@@ -240,3 +240,10 @@ async def finish():
         return 0
 
     return finish
+
+async def wait_for_view(cql: Session, name: str, node_count: int, timeout: int = 120):
+    async def view_is_built():
+        done = await cql.run_async(f"SELECT COUNT(*) FROM system_distributed.view_build_status WHERE status = 'SUCCESS' AND view_name = '{name}' ALLOW FILTERING")
+        return done[0][0] == node_count or None
```

Comment:
Really really weird that wait_for() requires you to return "None" if the condition hasn't yet happened. Makes the "view_is_built()" look very unnatural. In my opinion a better design would have required the pred function to throw if there's an error, and False would mean the condition is not **yet** ready.
Anyway, the design of the pre-existing `wait_for()` utility function is not your fault, of course :-)